services/frontend-service/src/ui/components/ReleaseCard/ReleaseCard.tsx

```diff
@@ -71,51 +71,78 @@ const rolloutStatusPriority = [
     RolloutStatus.ROLLOUT_STATUS_SUCCESFUL,
 ];
 
+const getRolloutStatusPriority = (status: RolloutStatus): number => {
+    const idx = rolloutStatusPriority.indexOf(status);
+    if (idx === -1) {
+        return rolloutStatusPriority.length;
+    }
+    return idx;
+};
+
+type DeploymentStatus = {
+    environmentGroup: string;
+    rolloutStatus: RolloutStatus;
+};
+
+const getAppRolloutStatus = (
+    status: StreamStatusResponse | undefined,
+    deployedVersion: number | undefined
+): RolloutStatus => {
+    if (status === undefined) {
+        // The status is completly unknown. Either the app is just new or rollout service is not responding.
```

Comment:
it would be nice if you could explain the different rollout statuses in the Readme.